redis/src/main/scala/zio/redis/options/Strings.scala

```diff
@@ -2,6 +2,38 @@ package zio.redis.options
 
 trait Strings {
 
+  sealed trait StrAlgoLCS {
+    self =>
+    private[redis] final def stringify: String =
+      self match {
+        case StralgoLCS.Strings => "STRINGS"
+        case StralgoLCS.Keys    => "KEYS"
+      }
+  }
+
+  object StralgoLCS {
+    case object Strings extends StrAlgoLCS
+    case object Keys    extends StrAlgoLCS
+  }
+
+  sealed trait StrAlgoLcsQueryType
+
+  object StrAlgoLcsQueryType {
+    case object Len                                                           extends StrAlgoLcsQueryType
+    case class Idx(minMatchLength: Int = 1, withMatchLength: Boolean = false) extends StrAlgoLcsQueryType
```

Comment:
It's not so clear from the documentation, but it looks like `MINMATCHLEN` can be specified without `IDX`.

Reply:
it indeed seems like you can, but it doesn;t seem to do anything

![image](https://user-images.githubusercontent.com/1788422/112437336-1e91d400-8d47-11eb-986c-181b0713b874.png)

Reply:
I agree, so let's keep it where it already is.